### PR TITLE
refactor and query for props

### PR DESCRIPTION
**Pull request by ogroppo**

### CODE REVIEW

File: src/lib/api.js [hunk 1]
```diff
@@ -15,36 +15,40 @@ export function search(term, language) {
...
-  ?item ?p ?statement.
-  ?claim wikibase:claim ?p.
-  ?claim wikibase:propertyType wikibase:WikibaseItem .
-  SERVICE wikibase:label { bd:serviceParam wikibase:language "en". }
-}`;
-  const url = wdk.sparqlQuery(query);
+export async function getItemProps(id, languageCode) {
+  const url = await new Promise(function (resolve, reject) {
+    try {
+      const query = `SELECT DISTINCT ?claim ?claimLabel WHERE {
```
Comment (ogroppo) [inline src/lib/api.js:21]: Do you think this query will do?

Comment (mshd) [reply src/lib/api.js:21]: Is it getting the inverse?

**Comment by mshd** [comment]
I still don't see necessity to fetch the all props and after that fetching the entity. But I checked it and it works now with properties which are linked to a given entity.
In lesser known languages, properties name's should fallback to English, now it just says "P141". The description should also be in the selected language then

File: src/lib/api.js (hunk 1 above)
Comment (ogroppo) [reply src/lib/api.js:21]: yes

**Comment by ogroppo** [comment]
which the description ? the one in search results? yes sure that is a todo

**Comment by ogroppo** [comment]
ok then we keep "en" fixed for the label service and if different we add the other lang so we can fall back?

**Pull request merged**